src/App.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/extensions */
+import React, { useEffect } from 'react';
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.  In this case, check that the `import/extensions` issue is fixed.